scripts/qa_crowbarsetup.sh

```diff
@@ -4608,7 +4608,10 @@ function onadmin_prepare_crowbar_upgrade()
         if crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
             complain 11 "Some repository is missing on admin server. Cannot upgrade."
         fi
-        safely crowbarctl upgrade prepare
+        # FIXME: crowbarctl command can time out easily
```

Comment:
The reason for this 16 days old PR is to have working version of upgrade prepare call, not to have another perfect solution.

The correct solution is indeed fixing the mentioned bug. Once it is fixed, we can return crowbarctl usage back.